BookAngular/src/app/book/listbook/listbook.component.ts

```diff
@@ -13,105 +13,105 @@ import { BookServiceService } from '../../services/book-service.service';
   templateUrl: './listbook.component.html',
   styleUrls: ['./listbook.component.scss']
 })
-export class ListbookComponent implements OnInit {
+  export class ListbookComponent implements OnInit {
 
-  
-  bookModel: BookModel;
-  booksDataGrid: any;
-  book: Book;
-  pagingModel: PagingModel = new PagingModel();
-  gridView: GridDataResult;
-  idUpDel: string;
-  searchString: string;
+    
+    bookModel: BookModel;
+    booksDataGrid: any;
+    book: Book;
+    pagingModel: PagingModel = new PagingModel();
+    gridView: GridDataResult;
+    idUpDel: string;
+    searchString: string;
 
-  // dialog field 
-  public isActive = false;
-  //check dialog create or edit
-  public isNew = false;
-  //loadBook toggle
-  public isReloadBooks = false;
-  public isActiveDeleteDialog = false;
-  public searchForm: FormGroup = new FormGroup({
-    searchString: new FormControl()
-  })
-  constructor(public service: BookServiceService
-    ) {}
-  
-  ngOnInit() {
-      this.loadBooks();
-      console.log("On init", this.isReloadBooks)
-      if(this.isReloadBooks)
-      {
+    // dialog field 
+    public isActive = false;
+    //check dialog create or edit
+    public isNew = false;
```

Comment:
The variable name should imply the meaning of the subject description, you should rename the variable and remove the comment.